ftw/simplelayout/mapblock/contents/configure.zcml

```diff
@@ -0,0 +1,22 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:five="http://namespaces.zope.org/five"
+    xmlns:genericsetup="http://namespaces.zope.org/genericsetup"
+    xmlns:i18n="http://namespaces.zope.org/i18n"
+    xmlns:zcml="http://namespaces.zope.org/zcml"
+    i18n_domain="ftw.simplelayout">
+
+    <!-- XXX Implement a specific KMLLayer for Simplelayout and do not recycle the one from geo.kml -->
+    <configure zcml:condition="installed collective.geo.kml">
```

Comment:
:scissors: `collective.geo.kml` is available because of `collective.geo.bundle` dependency.